### PR TITLE
chore(rxPageObjects): Refine doc comments and display

### PR DESCRIPTION
**Pull request by jj4th**
JIRA: https://jira.rax.io/browse/FRMW-1324

This is not a 1000 line PR, recommend viewing with the following link to remove whitespace only changes:
https://github.com/rackerlabs/encore-ui/pull/1982/files?w=1

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM
- [x] QE LGTM

Demo:
typedoc improvements

Things this PR does:
- Change some styles and remove oddly functioning search (TypeDoc problems, not ours).
- Hide inherited members which don't display well.
- Fix @see @link @example tags to display properly
- Remove @class and @namespace tags
- Fix a few OBVIOUS documentation mistakes I saw along the way (just two or three very light touches here).

### CODE REVIEW

File: utils/rx-page-objects/src/rxSearchBox.page.ts [hunk 1]
```diff
@@ -16,76 +13,84 @@ export class rxSearchBox extends rxComponentElement {
     }
 
     /**
-     * @description Getthe search value.
+     * Getthe search value.
```
Comment (Droogans) [inline utils/rx-page-objects/src/rxSearchBox.page.ts]: `Get the`

File: utils/rx-page-objects/src/rxSearchBox.page.ts [hunk 2]
```diff
@@ -16,76 +13,84 @@ export class rxSearchBox extends rxComponentElement {
...
+     *         expect(searchBox.getTerm()).to.eventually.equal('Some query');
+     *     });
      */
     getTerm(): Promise<string> {
         return this.txtSearch.getAttribute('value');
     }
 
     /**
-     * @description search for the given value.
+     * search for the given value.
```
Comment (Droogans) [inline utils/rx-page-objects/src/rxSearchBox.page.ts]: `Search`

File: utils/rx-page-objects/src/rxSelect.page.ts [hunk 3]
```diff
@@ -51,7 +48,7 @@ export class rxSelect extends rxComponentElement {
     }
 
     /**
-     * @description Whether the `<select>` element is valid.
+     * Whether the `<select>` element is valid.
```
Comment (Droogans) [inline utils/rx-page-objects/src/rxSelect.page.ts]: None of the other `select` keywords are expressed like this (`<select>`).

I think the tag style should be dropped.

File: utils/rx-page-objects/src/rxSelect.page.ts [hunk 4]
```diff
@@ -60,25 +57,29 @@ export class rxSelect extends rxComponentElement {
...
+     *         expect(dropdown.options.getText()).to.eventually.eql(texasLocations);
+     *     });
      */
     get options(): ElementArrayFinder {
         return this.$$('option');
     }
 
     /**
-     * @description The currently selected `<option>` element.
+     * The currently selected `<option>` element.
```
Comment (Droogans) [inline utils/rx-page-objects/src/rxSelect.page.ts]: `selected option element.`

File: utils/rx-page-objects/src/rxSortableColumn.page.ts [hunk 5]
```diff
@@ -9,71 +9,76 @@ export enum SORT_TYPE {
...
- *         var column = this.eleTableContainer.element(by.cssContainingText($('rx-sortable-column'), name));
- *         return new rxSortableColumn(column, 'item in repeaterString');
- *     }
- * };
+ *     var myTable = {
+ *         get eleTableContainer() { return $('#my-table'); },
+ *
+ *         column: function (name) {
+ *             var column = this.eleTableContainer.element(by.cssContainingText($('rx-sortable-column'), name));
+ *             return new rxSortableColumn(column, 'item in repeaterString');
```
Comment (Droogans) [inline utils/rx-page-objects/src/rxSortableColumn.page.ts]: Didn't the `repeaterString` arg get dropped?

File: utils/rx-page-objects/src/rxTimePicker.page.ts [hunk 6]
```diff
@@ -134,58 +134,58 @@ export class rxTimePicker extends rxComponentElement {
...
+     * Cancel picker, without updating value
      */
     cancel(): Promise<void> {
         return this.btnCancel.click();
     }
 }
 
 /**
- * @description return offset value, if present in string
+ * return offset value, if present in string
```
Comment (Droogans) [inline utils/rx-page-objects/src/rxTimePicker.page.ts]: `Return offset value`

File: utils/rx-page-objects/src/rxBulkSelect.page.ts [hunk 7]
```diff
@@ -110,7 +110,7 @@ export class rxBulkSelect extends rxComponentElement {
     }
 
     /**
-     * @description returns the row element by index.
+     * returns the row element by index.
```
Comment (Droogans) [inline utils/rx-page-objects/src/rxBulkSelect.page.ts]: Also, please capitalize the first word of the comment.

**Comment by jj4th** [comment]
Link to default themes for Steven: https://github.com/TypeStrong/typedoc-default-themes

File: utils/rx-page-objects/src/rxNotify.page.ts [hunk 8]
```diff
@@ -109,73 +115,81 @@ export class rxNotify extends rxComponentElement {
...
     }
 
     /**
-     * @description The number of notifications present in the scope of the namespace. For instance,
-     * calling this function on the object returned from {@link rxNotify.all} will inform you of the
-     * number of notifications present *anywhere* on the page. Calling it on {@link rxNotify.byStack}
-     * will return the count, but limited to the number of notifications in the stack selected.
+     * The number of notifications present in the scope of the factory created object.  Using this
+     * with [[rxNotify.all]] will yield the count of all the notifications on the page.  Using this
+     * with [[rxNotify.byStack('banner')]] will yield the count of all notifications in the `banner`
```
Comment (ssalinas24) [inline utils/rx-page-objects/src/rxNotify.page.ts]: Remove the Bracket Banner part.

File: utils/rx-page-objects/src/rxCopy.page.ts [hunk 9]
```diff
@@ -41,55 +38,57 @@ export class rxCopy extends rxComponentElement {
...
-     * @description Will click the "copy" icon. Attempts to copy the contents of the rxCopy instance
+     * Will click the "copy" icon. Attempts to copy the contents of the rxCopy instance
      * to the clipboard. You may experience issues attempting to do this in unsupported browsers.
+     *
      * @example
-     * var element = new encore.rxCopy($('.myCopyText'));
-     * element.copy();
+     *
+     *     var element = new encore.rxCopy($('.myCopyText'));
+     *     element.copy();
```
Comment (jj4th) [inline utils/rx-page-objects/src/rxCopy.page.ts]: so Typedoc would format it in a code block.

File: utils/rx-page-objects/doc-src/rx-theme/assets/css/extend.css [hunk 10]
```diff
@@ -0,0 +1,3 @@
+dd pre { clear: both; }
+dd pre code { margin-top: 1em; }
+.tsd-is-inherited { display: none !important; }
```
Comment (jj4th) [inline utils/rx-page-objects/doc-src/rx-theme/assets/css/extend.css]: Formatting the code block so it displays nicely below the `@example` tag.

Comment (jj4th) [inline utils/rx-page-objects/doc-src/rx-theme/assets/css/extend.css]: And hiding all "inherited" properties/methods.

File: utils/rx-page-objects/src/rxBulkSelect.page.ts (hunk 7 above)
Comment (jj4th) [inline utils/rx-page-objects/src/rxBulkSelect.page.ts]: This was a find and replace on `@description` I didn't change any of the docs unless it stuck out at me.

File: utils/rx-page-objects/src/rxActionMenu.page.ts [hunk 11]
```diff
@@ -3,7 +3,7 @@ import {by, ElementFinder} from 'protractor';
 import {OverrideWebdriver, Promise, rxComponentElement} from './rxComponent';
 
 /**
- * @description class to handle
+ * class to handle
```
Comment (jj4th) [inline utils/rx-page-objects/src/rxActionMenu.page.ts]: Not sure... I'd have to look at the old docs, I'll just rewrite this.

File: utils/rx-page-objects/src/rxMultiSelect.page.ts [hunk 12]
```diff
@@ -115,29 +114,32 @@ export class rxMultiSelect extends rxComponentElement {
...
     /**
-     * @description The option matching on the partial text in the option's name.
+     * The option matching on the partial text in the option's name.
+     *
      * @example
-     * var multiSelect = new encore.rxMultiSelect(element(by.model('cars')));
-     * var option = multiSelect.option('Ford Bronco');
-     * option.select();
-     * option.deselect();
-     * @see rxCheckbox
```
Comment (jj4th) [inline utils/rx-page-objects/src/rxMultiSelect.page.ts]: I'll look at this, it's possible I removed this intending to roll it into the description.  I was just find/replacing these with `@see [[link]]`

File: utils/rx-page-objects/src/rxMultiSelect.page.ts [hunk 13]
```diff
@@ -147,15 +149,17 @@ export class rxMultiSelect extends rxComponentElement {
     }
 
     /**
-     * @description Given a list of options, deselect each of them. Will not
+     * Given a list of options, deselect each of them. Will not
+     *
```
Comment (jj4th) [inline utils/rx-page-objects/src/rxMultiSelect.page.ts]: Probably, again, not something I changed in this PR.  I'll fix it.

File: utils/rx-page-objects/src/rxMultiSelect.page.ts (hunk 12 above)
Comment (jj4th) [inline utils/rx-page-objects/src/rxMultiSelect.page.ts]: Yeah the entire doc for this method is wrong, it no longer returns a wrapped element.

Comment (jj4th) [inline utils/rx-page-objects/src/rxMultiSelect.page.ts]: But it probably should return a wrapped element.

**Comment by cathysiller** [comment]
Dev LGTM

**Comment by jj4th** [comment]
@Droogans @CITguy feedback addressed.

**Pull request merged**